mico-admin/src/app/app-detail/app-detail.component.ts

```diff
@@ -38,6 +38,7 @@ export class AppDetailComponent implements OnInit, OnDestroy {
     selectedVersion;
     allVersions;
     publicIps: string[] = [];
+    isLatestVersion: boolean;
```

Comment:
make isLatestVersion a calculated property (with a getter function) to avoid missing updating this variable.